docs/distributions/fedora/home.md

```diff
@@ -1,3 +1,3 @@
 Welcome to the T2 Fedora wiki!
 
-Click [here](https://wiki.t2linux.org/distributions/fedora/installation/) to get started.
+Follow [the installation guide](https://wiki.t2linux.org/distributions/fedora/installation/) to get started. If you need help, join the T2Linux discord [server](https://discord.com/invite/68MRhQu) server and ask in `#fedora`. Remember to read the FAQ [here](https://wiki.t2linux.org/distributions/fedora/faq/) to fix common problems.
```

Comment:
This looks nice. Can you do this for Ubuntu as well.

Also, *You can also refer to* the FAQ to fix common problems.